### PR TITLE
Deprecate old GT RA within GT++

### PR DESCRIPTION
**Pull request by chochem**
please use RA2.

### CODE REVIEW

**Review by BlueWeabo — APPROVED** [review]
I agree

File: src/main/java/gtPlusPlus/xmod/gregtech/api/interfaces/internal/IGregtech_RecipeAdder.java [hunk 1]
```diff
@@ -336,8 +374,10 @@ boolean addColdTrapRecipe(int aCircuit, ItemStack aInput, FluidStack aFluidInput
     boolean addReactorProcessingUnitRecipe(ItemStack aInput1, ItemStack aInput2, FluidStack aFluidInput,
             ItemStack[] aOutputs, int[] aChances, FluidStack aFluidOutput, int aTime, int aEU);
 
+    @Deprecated
     boolean addFluidHeaterRecipe(ItemStack aCircuit, FluidStack aInput, FluidStack aOutput, int aDuration, int aEUt);
 
+    @Deprecated
     boolean addVacuumFreezerRecipe(ItemStack aInput, ItemStack aOutput, int aDuration, int aEU);
 
     boolean addMolecularTransformerRecipe(ItemStack aInput, ItemStack aOutput, int aDuration, int aEU);
```
Comment (BlueWeabo) [inline src/main/java/gtPlusPlus/xmod/gregtech/api/interfaces/internal/IGregtech_RecipeAdder.java:383]: you missed one

**Review by BlueWeabo — CHANGES_REQUESTED** [review]

**Comment by boubou19** [comment]
OMFG a straight copy of the GT5U interface 😭

File: src/main/java/gtPlusPlus/xmod/gregtech/api/interfaces/internal/IGregtech_RecipeAdder.java (hunk 1 above)
Comment (chochem) [reply src/main/java/gtPlusPlus/xmod/gregtech/api/interfaces/internal/IGregtech_RecipeAdder.java:383]: molecular transformer multi is a gtpp recipe map. I dont think those have RA2. correct me if I am wrong

**Review by BlueWeabo — APPROVED** [review]

**Comment by boubou19** [comment]
503 on setting up the job task, i guess this PR can't be merged for now

**Comment by chochem** [comment]
rerun did the trick

**Pull request merged**